pkgs/by-name/sh/shtris/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "shtris";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "ContentsViewer";
+    repo = "shtris";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-lAuDM6461fR+i3KkgcCcytRT6llSj0kEoqK6N8Q3kVI=";
+  };
+
+  dontConfigure = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm555 shtris $out/bin/shtris
+
+    runHook postInstall
+  '';
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = [ "--version" ];
+  doInstallCheck = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "The pure shell script (sh) that implements the Tetris game following the Tetris Guideline (2009)";
```

Comment:
The description should not start with `The` or `A`.
(See https://nixos.org/manual/nixpkgs/stable/#var-meta-description)